test/integration/connect/envoy/docker-compose.yml

```diff
@@ -14,667 +14,37 @@ volumes:
 services:
```

Comment:
Note: I didn't delete this file yet because I forgot to move the ad-hoc tcpdump stuff over.